index.html

```diff
@@ -35,6 +36,99 @@ <h2>
             <li><img class="M" src="Images/M.png" alt="M">
         </ul>
     </main>
-    
+    </div>
+    <div class="Works">
+        <article>
+            <h2 class="First">My Recent Works</h2>
```

Comment:
Please be sure to follow as much as possible the design on Figma. Following the proper fonts as the design is super important 🤓 Your heading for 'My Recent Works' doesn't have the proper font.